src/types/index.ts

```diff
@@ -38,7 +38,7 @@ export * from "./workbook_data";
 
 declare global {
   interface Window {
-    Chart: typeof Chart;
+    Chart: typeof Chart & typeof Chart.Chart;
```

Comment:
window.Chart is both the class `Chart`, and all the exports of chartJS lib. A bit strange but 🤷